ui/main/src/app/modules/cards/components/detail/detail.component.ts

```diff
@@ -170,7 +170,7 @@ export class DetailComponent implements OnChanges, OnInit, OnDestroy, AfterViewC
     }
 
     open(content) {
-        const modalOptions = { windowClass : 'opfab-modal-content'};
+        const modalOptions = { windowClass : 'opfab-modal-content', centered: true};
```

Comment:
The popup for "delete card" is slightly shifted to the left.Can you modify please ?
The 3 other popups are centered.